src/Psalm/Internal/FileManipulation/FunctionDocblockManipulator.php

```diff
@@ -229,7 +229,7 @@ private function __construct(string $file_path, FunctionLike $stmt, ProjectAnaly
                 continue;
             }
 
-            if ($chars[$i] === '\\' || preg_match('/\w/', $char)) {
+            if ($char === '\\' || preg_match('/\w/', $char)) {
```

Comment:
@orklah are you sure it's the same as `$chars[$i]`? It looks like `$i` is changed above, after `$char` assignment.

Reply:
I think so, every change to $i is followed by a continue; but I agree it's confusing, I'll revert that